paperscraper/lib.py

```diff
@@ -288,7 +301,7 @@ def default_scraper(
     return scraper
 
 
-def parse_semantic_scholar_metadata(paper: dict[str, Any]) -> dict[str, Any]:
+async def parse_semantic_scholar_metadata(paper: dict[str, Any]) -> dict[str, Any]:
```

Comment:
Just wondering, why make this `async`? Is it just to match the signature of `parse_google_scholar_metadata`?

It's a CPU bound function with no `await`s, so imo this shouldn't be `async`

Reply:
These functions are passed later on as args and I need them to both be awaitable for simplicity